website/docs/components/form/radio-card/partials/guidelines/guidelines.md

```diff
@@ -55,6 +55,38 @@ In most cases, we recommend using the bottom position. Still, we offer Radio Car
   </G.RadioCard>
 </Hds::Form::RadioCard::Group>
 
+## Nested badge
+
+Badges can be used in radio cards to display additional information and context. To ensure proper usage of the badge component, refer to the [guidelines.](/components/badge) 
+
+<Hds::Form::RadioCard::Group @name="radio-card-custom-example" as |G|>
```

Comment:
```suggestion
<Hds::Form::RadioCard::Group @name="radio-card-nested-badge" as |G|>
```

Issue: Because there is another set of radio elements with the same name `radio-card-custom-example` on this page (in the 'Code' tab) the control looses selection, while the card remains styled as selected.

<img width="390" alt="Screenshot 2023-04-28 at 09 49 18" src="https://user-images.githubusercontent.com/788096/235101443-145e0033-9a60-4d2a-8e3f-333f3d4ce20f.png">

To avoid this, we must provide a unique name to each `RadioCard::Group`